client/src/App.tsx

```diff
@@ -18,14 +18,11 @@ function App(): JSX.Element {
       <Switch>
         
         <Route path='/signup' strict component={SignUpModal}></Route>
-        <Route path='/confirmation' component={() => <ConfirmationModal email="anna@pregnancycentre.ca"/>}></Route>
```

Comment:
the changes to App.tsx removes unused endpoints that were added for testing purposes when building the auth UI